src/cljs/webchange/ui_framework/layout/navigation/views.cljs

```diff
@@ -23,33 +44,33 @@
         handle-click #(goto props)]
     [:li {:class-name (get-class-name {"menu-item" true
                                        "active"    active?})}
-     [button {:on-click handle-click
-              :color    "default"
-              :variant  "outlined"}
+     [:span
+      [user-icn/data]]
+     [:span   {:on-click handle-click}
       title]
      (when-not (empty? children)
        [menu {:items children}])]))
 
 (defn- menu
   [{:keys [items]}]
-  [:ul.menu
+  [:ul
    (for [{:keys [title] :as props} items]
      ^{:key title}
      [menu-item props])])
 
 (defn navigation-menu
   []
-  (let [current-course-id @(re-frame/subscribe [::state/current-course-id])
-        menu-items [{:title         "My Profile"
-                     :location-name :profile}
-                    (cond-> {:title         "Courses"
-                             :location-name :courses}
-                            (some? current-course-id) (assoc :children [{:title        "Current course"
-                                                                         :route-name   :course-editor-v2
-                                                                         :route-params [:id current-course-id]}]))
-                    {:title         "Books"
-                     :location-name :books}
-                    {:title      "Create new book"
-                     :route-name :book-creator}]]
-    [:div.navigation-menu
-     [menu {:items menu-items}]]))
+  (let [current-user @(re-frame/subscribe [::as/user])
+        current-course-id @(re-frame/subscribe [::state/current-course-id])]
+    [:div.left-side-bar
+     [:div.pad-fit-l.mb-lg.d-inline
+      [logo]]
+     [:div.nav-menu
+      [menu {:items menu-items}]]
+     [:div.g-icon-bottom
+      [:span
+       [avatar]]
+      [:span
+       (str (:first-name current-user) " " (:last-name current-user))
+       [:small 
+        "Los Angeles, CA, USA"]]]]))
```

Comment:
I don't think we need hard-coded location. Remove, please. After remove do not forget to check vertical alignment relative to logo.